Assets/Mirror/Editor/NetworkManagerEditor.cs

```diff
@@ -91,9 +80,6 @@ protected void Init()
 
             // network foldout properties
             networkAddressProperty = serializedObject.FindProperty("networkAddress");
```

Comment:
Do we need a foldout for just 1 property?

Reply:
there are two: address and max connections.
the question is if we need foldouts at all. might be something for another pr though.